cves/CVE-2017-5651.yml

```diff
@@ -43,7 +43,13 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  In certain versions of Apache Tomcat, the refactoring of the HTTP
+  connectors introduced a regression in send file processing. If the processing
```

Comment:
"the refactoring of the HTTP connectors introduced a regression in send file processing" could be rewritten in easier to understand language.

Reply:
I second this